src/globus_cli/parsing/param_types/comma_delimited.py

```diff
@@ -16,6 +18,9 @@ def __init__(
         self.convert_values = convert_values
         self.choices = list(choices) if choices is not None else None
 
+    def get_type_annotation(self, param: click.Parameter) -> type:
+        return list[str]
```

Comment:
Are the elements in a `CommaDelimitedList` always strings?

Reply:
Yes, we're guaranteed it. `convert_values` looks like it could modify things, but it's typed as a `str -> str` function.